### PR TITLE
great new feature

### PR DESCRIPTION
**Pull request by hammerdj1**
I think you'll like this!!!

### CODE REVIEW

File: src/main/java/com/inmarsat/demo/ContactsExampleRoute.java [hunk 1]
```diff
@@ -50,6 +50,14 @@ public ContactsExampleRoute() {
     public void configure() throws Exception {
         // you can configure the route rule with Java DSL here
 
+        String unused = "Unused Var";
```
Comment (cloudbotinmarsat) [inline src/main/java/com/inmarsat/demo/ContactsExampleRoute.java]: MAJOR Remove this unused "unused" local variable. [rule](http://172.31.16.164:9000/coding_rules#rule_key=squid%3AS1481)
MAJOR Remove this useless assignment to local variable "unused". [rule](http://172.31.16.164:9000/coding_rules#rule_key=squid%3AS1854)

File: src/main/java/com/inmarsat/demo/ContactsExampleRoute.java [hunk 2]
```diff
@@ -50,6 +50,14 @@ public ContactsExampleRoute() {
...
+
+        try {
```
Comment (cloudbotinmarsat) [inline src/main/java/com/inmarsat/demo/ContactsExampleRoute.java]: MAJOR Either remove or fill this block of code. [rule](http://172.31.16.164:9000/coding_rules#rule_key=squid%3AS00108)

File: src/main/java/com/inmarsat/demo/ContactsExampleRoute.java [hunk 3]
```diff
@@ -50,6 +50,14 @@ public ContactsExampleRoute() {
...
+
+        } catch(Exception e) {
```
Comment (cloudbotinmarsat) [inline src/main/java/com/inmarsat/demo/ContactsExampleRoute.java]: MAJOR Either log or rethrow this exception. [rule](http://172.31.16.164:9000/coding_rules#rule_key=squid%3AS1166)
MAJOR Either remove or fill this block of code. [rule](http://172.31.16.164:9000/coding_rules#rule_key=squid%3AS00108)

**Comment by cloudbotinmarsat** [comment]
SonarQube analysis reported 5 issues
* MAJOR 5 major

Watch the comments in this conversation to review them.

File: src/main/java/com/inmarsat/demo/ContactsExampleRoute.java (hunk 3 above)
Comment (itcloudservices) [inline src/main/java/com/inmarsat/demo/ContactsExampleRoute.java]: THis is a bit odd.

**Pull request closed without merging**